src/routes/_actions/timeline.js

```diff
@@ -88,6 +89,9 @@ async function fetchTimelineItemsAndPossiblyFallBack () {
     lastTimelineItemId,
     online
   } = store.get()
+  if (currentInstance === undefined || currentInstance === null) {
+    currentInstance = await registerDefaultInstance()
+  }
```

Comment:
It makes me a bit uncomfortable that we need to put these checks in so many different places. I imagine we must be missing a few (e.g. the search page). Can we not put a string, `defaultInstance` into `store.js` and look for that instead?